### PR TITLE
Add UoExeter 1-1-3

### PR DESCRIPTION
**Pull request by znichollscr**
## Checklist

Please confirm that this pull request has done the following:

- [x] Documentation added (where applicable)
- [x] Changelog item added to `changelog/`

### CODE REVIEW

**Comment by durack1** [comment]
Looks to me like this has picked up a bunch of `CR-CMIP-0-3-0` changes?

File: CVs/input4MIPs_source_id.json [hunk 1]
```diff
@@ -121,5 +121,13 @@
         "license_id":"CC BY 4.0",
         "mip_era":"CMIP6Plus",
         "source_version":"1.1.2"
+    },
```
Comment (durack1) [inline CVs/input4MIPs_source_id.json:124]: Just noting that we need to either delete or deprecate the `UOEXETER-CMIP-1-1-2` which never really existed (same with `UOEXETER-CMIP-0-1-0`)

**Comment by znichollscr** [comment]
> Looks to me like this has picked up a bunch of CR-CMIP-0-3-0 changes?

Not quite. I hadn't run the validation on those files. That's the change that's picked up. (So the files haven't changed, but our validation status of them has, which is the changes you're seeing)

File: python-packages/input4MIPs-CVs/src/input4MIPs_CVs/cli/update-database.py [hunk 2]
```diff
@@ -217,11 +217,11 @@ def other_manual_fixes(db_df: pd.DataFrame) -> pd.DataFrame:
     out.loc[out["source_id"] == "UOEXETER-CMIP-0-1-0", "publication_status"] = (
         "never_published"
     )
-
-    # Source IDs in publishing queue
     out.loc[out["source_id"] == "UOEXETER-CMIP-1-1-2", "publication_status"] = (
-        "in_publishing_queue"
```
Comment (durack1) [inline python-packages/input4MIPs-CVs/src/input4MIPs_CVs/cli/update-database.py:223]: Ok this means we can ignore my earlier comment

**Review by durack1 — APPROVED** [review]

**Comment by durack1** [comment]
Ok and just checked the 12 pm scrape includes 7 UOEXETER-CMIP-1-1-3 files, and it does!

**Pull request merged**